src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -329,19 +327,24 @@ export class MeetingManager implements AudioVideoObserver {
       console.log('[MeetingManager audioVideoDidStop] Meeting ended for all');
       this.meetingStatus = MeetingStatus.Ended;
       this.publishMeetingStatus();
+      this.leave();
```

Comment:
Can you put a comment here explain why we do not call leave here?
Also maybe a switch statement is better here :D

Reply:
we do call leave() for this case - and yes, Im swithcing to use a swtich case